source/css/patternfly.css

```diff
@@ -6092,8 +6092,15 @@ tbody.collapse.in {
       display: inline-block;
       width: 100%; } }
 
-.card {
-  box-shadow: 0 0.1em 0.2em rgba(3, 3, 3, 0.1); }
+.pf-card::after {
```

Comment:
The shadow is applied to `::after` because otherwise the selectable card border (when transparent) looked bad when combined with the box-shadow and header bottom border.